MANUAL.adoc

```diff
@@ -849,6 +849,27 @@ Most constants can be created once and reused as many times as needed.
 [id=const]
 ==== The `Const` interface
 
+The `Const` interface is used to represent expressions which are constant.
+This interface also contains factory methods to create new constants, including (but not limited to):
+
+* Primitive values (e.g. integers, booleans)
+* Primitive wrapper type instances (e.g. `Long`, `Double`)
```

Comment:
I don't think we have that? The factory methods take wrapper types (such as `Boolean`), but always produce primitive constants (such as `boolean`).